src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts

```diff
@@ -53,11 +53,11 @@ export class MultiAlertService {
     }
 
     next() {
-        this.current = (this.current === this.activeAlerts.length - 1) ? this.current : this.current + 1;
+        this.current = (this.current === this.activeAlerts.length - 1) ? 0 : this.current + 1;
```

Comment:
🤔  Is cycling alerts a good behavior though? What If it has only one alert? When there is only alert, users can still keep clicking on both previous and next buttons?

Reply:
If there is only one alert, the cycle control should be showing.

Reply:
The cycle control should also disappear after the second to last alert is dismissed, then leaving only one alert

Reply:
As @reddolan correctly pointed out, if there is only one alert, the pager disappears.

Reply:
Oh so the pager disappears. Thanks for the clarification.